waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -135,7 +135,6 @@ npmDepsForWasp _spec =
             ("typescript", "^5.1.0"),
             ("@types/react", "^18.0.37"),
             ("@types/react-dom", "^18.0.11"),
-            ("@types/react-router-dom", "^5.3.3"),
```

Comment:
Types are not included in the package, there is no need for extra `@types` package.

Reply:
What do you mean by this? Where are they not included?

Reply:
My bad, I miswrote: types _are_ included in the package.